build.gradle.kts

```diff
@@ -204,7 +204,8 @@ tasks.register("detektAll") {
         ":dd-sdk-android-sqldelight:detektMain",
         ":dd-sdk-android-timber:detektMain",
         ":instrumented:integration:detekt",
-        ":instrumented:nightly-tests:detekt",
+        ":instrumented:nightly-tests:detektMain",
```

Comment:
Just for the note: It seems this change partially overlaps with what is described in https://datadoghq.atlassian.net/browse/RUMM-1882

Reply:
I see, maybe we should not apply custom detekt rules for `nightly-tests` module. @xgouchet what do you think ?